src/Test/PdbUtilities/Metadata/ILVisualizer.cs

```diff
@@ -285,11 +285,27 @@ public void VisualizeHeader(StringBuilder sb, int codeSize, int maxStack, Immuta
             }
 
             int spanIndex = 0;
-            int curIndex = DumpILBlock(ilBytes, length, sb, spans, blockOffset, 0, spanIndex, IndentString, markers, out spanIndex);
+            string markerPadding = GetMarkerPadding(markers.Values);
+            int curIndex = DumpILBlock(ilBytes, length, sb, spans, blockOffset, 0, spanIndex, markerPadding, markers, out spanIndex);
             Debug.Assert(curIndex == length);
             Debug.Assert(spans == null || spanIndex == spans.Count);
         }
 
+        private string GetMarkerPadding(IEnumerable<string> values)
+        {
+            int max = 0;
+            foreach (var v in values)
+            {
+                max = Math.Max(max, v.Length);
+            }
+            string pad = " ";
```

Comment:
>string pad = " " [](start = 12, length = 16)

I think there is a constructor that can be used to replace this line and the ```for``` below.